prow/config/config.go

```diff
@@ -823,6 +827,13 @@ func parseProwConfig(c *Config) error {
 		c.Deck.Spyglass.RegexCache[k] = r
 	}
 
+	if c.Deck.DefaultOrg == "" {
```

Comment:
probably worth noting that this is not specific to this location, this is an ongoing compatibility thing. otherwise anyone reading this will question why this block exists :-)
(and they still will)

Reply:
(or removing it, and requiring that it be in config explicitly...)

Reply:
I decided to use the Plank default decoration to provide these fields, instead of adding more config that should really be the same thing